sscanss/app/window/view.py

```diff
@@ -742,6 +750,17 @@ def showPreferences(self, group=None):
         preferences.setModal(True)
         preferences.show()
 
+    def showCurveEditor(self):
+        """Opens the volume curve editor dialog"""
+        volume = self.presenter.model.volume
+        if volume is None:
+            self.showMessage('No volume has been added to the project.', MessageType.Information)
+            return
+
+        curve_editor = CurveEditor(volume, self)
+        curve_editor.setModal(True)
```

Comment:
If this was not modal, would it be possible to rotate the 3d main view while the dialog was open? That could be useful.

Reply:
Yes its possible, I wanted the dialog to always stay on top but still give the user access to rotate the 3d view but I could not get  Qt::WindowStaysOnTopHint to work on Linux. I planned to revisit it later but for now I will set modal to False